src/extensions/renderer/base/arrow-shapes.js

```diff
@@ -232,15 +220,10 @@ BRp.registerArrowShapes = function(){
     },
 
     draw: function( context, size, angle, translation, edgeWidth ){
-      var scaledCrossLine = [
-        this.crossLinePoints[0] + this.scaleCrossLineXCoord( size, edgeWidth ),
-        this.crossLinePoints[1] - this.scaleCrossLineYCoord( size, edgeWidth ),
-        this.crossLinePoints[2] - this.scaleCrossLineXCoord( size, edgeWidth ),
-        this.crossLinePoints[3] - this.scaleCrossLineYCoord( size, edgeWidth )
-      ];
+      var scaledCrossLineSize = size*0.94 + edgeWidth + 2; 
```

Comment:
Not sure about this.  The alignment is a bit off for thick edges with small arrows:

![screen shot 2017-08-24 at 2 14 44 pm](https://user-images.githubusercontent.com/989043/29681722-9b517460-88d6-11e7-969a-5bbbe231cdc1.png)